wasm-runtime/polkadot/src/support/storable.rs

```diff
@@ -17,29 +17,48 @@
 //! Stuff to do with the runtime's storage.
 
 use slicable::Slicable;
-use endiansensitive::EndianSensitive;
 use keyedvec::KeyedVec;
 use runtime_support::prelude::*;
 use runtime_support::{self, twox_128};
 
 /// Trait for a value which may be stored in the storage DB.
 pub trait Storable {
 	/// Lookup the value in storage and deserialise, giving a default value if not found.
-	fn lookup_default(key: &[u8]) -> Self where Self: Sized + Default { Self::lookup(key).unwrap_or_else(Default::default) }
+	fn lookup_default(key: &[u8]) -> Self where Self: Sized + Default {
+		Self::lookup(key).unwrap_or_else(Default::default)
+	}
+
 	/// Lookup `Some` value in storage and deserialise; `None` if it's not there.
-	fn lookup(_key: &[u8]) -> Option<Self> where Self: Sized { unimplemented!() }
+	fn lookup(_key: &[u8]) -> Option<Self> where Self: Sized {
+		unimplemented!()
```

Comment:
having a default trait function be `unimplemented!` is strange. why not just leave it unimplemented?

Reply:
:+1: We had that in RPC some time ago and it caused some of the RPC methods to be unintentionally unimplemented because of that.

Reply:
because this is not always implementable (i.e. when `Self` is not `Sized`). in that case, i can't leave it unimplemented but also can't implement it. having a default implementation is the only way around this i found. happy to change if there's a cleaner way.

Reply:
(point is moot now with the new storage API)